tests/Integration/inc/Engine/Optimization/TestCase.php

```diff
@@ -1,14 +1,44 @@
 <?php
 
-
 namespace WP_Rocket\Tests\Integration\inc\Engine\Optimization;
 
 use WP_Rocket\Tests\Integration\FilesystemTestCase;
 
 abstract class TestCase extends FilesystemTestCase {
-	protected $cnames;
-	protected $zones;
-	protected   $settings;
+	protected        $cnames;
+	protected        $zones;
+	protected        $settings;
+	protected static $original_settings;
+	protected        $old_settings = [];
+
+	public static function setUpBeforeClass() {
+		parent::setUpBeforeClass();
+
+		self::$original_settings = (array) get_option( 'wp_rocket_settings', [] );
+	}
+
+	public static function tearDownAfterClass() {
+		parent::setUpBeforeClass();
+
+		update_option( 'wp_rocket_settings', self::$original_settings );
+	}
+
+	public function setUp() {
+		$this->wp_content_dir = 'vfs://public/wp-content';
```

Comment:
is this still necessary when using the FilesystemTestCase?

Reply:
Nope. It was removed.